data/json/vehicle_parts.json

```diff
@@ -5104,5 +5104,81 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "count": [1, 2]}
         ]
+    },{
+        "type" : "vehicle_part",
+        "id" : "plow",
+        "name" : "Plow",
+        "symbol" : "&",
+        "color" : "blue",
+        "broken_symbol" : "*",
+        "broken_color" : "light_blue",
+        "damage_modifier" : 40,
+        "durability" : 200,
+        "item" : "sheet_metal",
+        "difficulty" : 4,
+        "location" : "under",
+        "bonus": 150,
```

Comment:
Test case:
Add a plow to a beetle, activate it, start driving.
Expected result is the beetle stalling and being unable to move.
Actual result, it's not straining the engine until it hits 60MPH.
With a regular car, you can add 4x plows and plow at 36MPH
Finally, you can attach a plow to a bicycle and plow at above regular walking speed without getting tired.
I don't know if just adjusting the bonus value will do the right thing, but it's totally broken right now.